models/transformation/dme_claims.sql

```diff
@@ -3,7 +3,7 @@ select
     , cast(clm_line_num as integer) as claim_line_number
     , 'professional' as claim_type
     , {{ cast_string_or_varchar('bene_mbi_id') }} as patient_id

```

Comment:
Should we remove this unnecessary custom macro since you found a built-in function to use?

Reply:
we definitely should... theres so much of it throughout the project that Im going to save that for another PR